extract_moves_test.py

```diff
@@ -0,0 +1,35 @@
+import re
+import unittest
+
+from data_compressor import extract_moves
+
+path = ...  # path to pgn with games
+
+
+class MyTestCase(unittest.TestCase):
+    """Tests for function ``data_comressor.extract_moves``
+    """
+
+    def test_moves_number(self):
```

Comment:
For example test pgn file with n games and k1, k2, ..., kn moves each, you can check if all the moves were extracted